### PR TITLE
Photo functionality enabled

### PR DESCRIPTION
**Pull request by petrakh**

### CODE REVIEW

**Review by BDCraven — COMMENTED** [review]
:shipit:

**Review by EOSullivanBerlin — APPROVED** [review]

File: spec/features/create_photos_spec.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+feature 'Photo' do
+  it 'can create a photo' do
+    visit 'photos/new'
+    attach_file('photo_image', Rails.root + 'spec/files/test.png', visible: false)
+    click_button 'Upload'
+    expect(Photo.count).to eq 1
+  end
```
Comment (dbugsy) [inline spec/features/create_photos_spec.rb:7]: check the bad path as well - what if a user tries to upload something that is not a photo? Are there any edge cases you can think of?

File: spec/features/create_photos_spec.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+feature 'Photo' do
+  it 'can create a photo' do
+    visit 'photos/new'
+    attach_file('photo_image', Rails.root + 'spec/files/test.png', visible: false)
+    click_button 'Upload'
+    expect(Photo.count).to eq 1
```
Comment (dbugsy) [inline spec/features/create_photos_spec.rb:6]: small code smell here: checking the exact count is brittle. You can get around it by counting the current photos at the beginning of the test and then expecting that the count increases.

**Review by dbugsy — APPROVED** [review]
:shipit: :shipit: I am fine with this PR, the comments are more for information than actually asking for change.

**Pull request merged**